js/src/modal.js

```diff
@@ -63,7 +65,9 @@ const Selector = {
   DATA_TOGGLE    : '[data-toggle="modal"]',
   DATA_DISMISS   : '[data-dismiss="modal"]',
   FIXED_CONTENT  : '.fixed-top, .fixed-bottom, .is-fixed, .sticky-top',
-  STICKY_CONTENT : '.sticky-top'
+  STICKY_CONTENT : '.sticky-top',
+  FOCUSABLE      : 'a[href], area[href], input, button, select, textarea, ' +
+    'audio[controls], video[controls], iframe, object, embed, [tabindex], [contenteditable]'
```

Comment:
would be good, if possible, to exclude things with `tabindex` lower than `0` (e.g. `tabindex="-1"`)

Reply:
And should probably filter the elements to make sure they are visible (i.e. not `display: none`, etc)

Reply:
JQuery has a visible selector, and in vanillaJs you can just check that the element's `boundingClientRect` height and width are greater than `0`

Reply:
Fixed.

Simply replacing the `:visible` selecter with `boundingClientRect` achieved poor performance with Chrome (https://jsperf.com/finding-tabbable-elements/24). So, `offsetWidth` is also added as jQuery does.